src/services/WorkflowHandler.ts

```diff
@@ -59,21 +60,33 @@ function processSteps(inputData: InputData | AIMessage): { stepsArray: string[][
   return { stepsArray, fullPlan };
 }
 
-
-function processResults(results: any | AIMessage): string {
+function processResults(results: any | AIMessage): any {
```

Comment:
Type this, create new type in types file:
interface ErrorResponse {
  error: string;
}
And update any to ErrorResponse | Record<string, unknown>